docs/extensions/readme.md

```diff
@@ -996,6 +997,45 @@ Full code:
 [example3.cs](x-ms-pageable/example3.cs)
 
 
+**Example 4: Passing the continuation token through an API parameter**
```

Comment:
Do we need a sample that has operationName  + tokenParamName?

Reply:
I don't think we need both at this time, and I wanted to keep it simpler (and also wanted to write less), so I went with not defining a separate next operation. I do see where it should be more clear that this will also work with a separate next operation, I'll add that to the `tokenParamName` description